### PR TITLE
Update access-control.yaml after the Atala team restructure

### PR DESCRIPTION
**Pull request by yshyn-iohk**

### CODE REVIEW

File: access-control.yaml [hunk 1]
```diff
@@ -1387,8 +1362,7 @@ teams:
       - baziorek
       - berendsliedrecht
       - bestbeforetoday
-      - bgravenorst
-      - blockchaintimothy
+      - bgravenors
```
Comment (FabioPinheiro) [inline access-control.yaml:1365]: missing a `t`

**Comment by FabioPinheiro** [comment]
Any one you remove from `identus-admins` should go to `identus-maintainers`?
I wouldn't change anything in `identus-maintainers` for now

File: access-control.yaml [hunk 2]
```diff
@@ -1503,22 +1474,18 @@ teams:
       - mastersingh24
       - matkt
       - matthew1001
-      - mattklepp
       - maurges
       - mayktu
       - mbaxter
-      - mbrandenburger
```
Comment (ryjones) [inline access-control.yaml:1510]: why

**Review by ryjones — CHANGES_REQUESTED** [review]
This seems fairly sloppy. please carefully review your changes.

**Comment by tkuhrt** [comment]
It would be great if you can explain this change. This seems like you are completely restructuring maintainers of the project. We will need all existing maintainers to weigh in on these changes.

File: access-control.yaml (hunk 2 above)
Comment (yshyn-iohk) [reply access-control.yaml]: Sorry, my bad
thanks for fixing

**Comment by yshyn-iohk** [comment]
I replaced `essbante-io` GitHub id with mine in the identus-maintainers team.
All other people were let go when the Atala Tribe was restructured:
- CryptoKnightIOG
- EzequielPostan
- FayyaadhAdams
- blockchaintimothy
- cristianIOHK
- curtis-h
- lohanspies
- milosbackonja
- milosh86
- mkbreuningIOHK
- patlo-iog
- petevielhaber
- shotexa
- womfoo
- Dale-iohk
- urosmrvic-iohk
- jesusdiazvico
- mattklepp

File: access-control.yaml (hunk 1 above)
Comment (yshyn-iohk) [reply access-control.yaml]: Thanks, @FabioPinheiro. My bad.

**Comment by yshyn-iohk** [comment]
`bgravenorst` - was touched by me unintentionally

**Comment by yshyn-iohk** [comment]
https://github.com/hyperledger/governance/pull/336 is the new PR according to the changes in the Atala

**Pull request closed without merging**